src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig

```diff
@@ -1,3 +1,4 @@
+{% deprecated 'The template "' ~ _self ~'" is deprecated since Symfony 4.4, will be removed in 5.0.' %}
 {% extends '@Twig/layout.html.twig' %}
```

Comment:
The `layout.html.twig` can also be deprecated as it's only used by the deprecated exception pages. Same applies to `base_js.html.twig` and `exception.css.twig`.